genestack_client/scripts/genestack_application_manager.py

```diff
@@ -416,10 +419,17 @@ def mark_as_stable(application, version, app_id_list, scope):
 
 def remove_applications(application, version, app_id_list):
     print('Removing application(s) with version "%s"' % version)
-    for app_id in app_id_list:
-        sys.stdout.write('%-40s ... ' % app_id)
+    if app_id_list is not None:
+        for app_id in app_id_list:
+            sys.stdout.write('%-40s ... ' % app_id)
+            sys.stdout.flush()
+            application.invoke('removeApplication', app_id, version)
+            sys.stdout.write('ok\n')
+            sys.stdout.flush()
+    else:
+        sys.stdout.write('ALL ... ')
```

Comment:
I would really prefer to see list of apps being removed.  Or been removed.
Maybe `removeApplications()` could return the list of app ids that were deleted?